src/Sulu/Bundle/TestBundle/Testing/AssertHttpStatusCodeTrait.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TestBundle\Testing;
+
+use Symfony\Component\HttpFoundation\Response;
+
+trait AssertHttpStatusCodeTrait
+{
+    /**
+     * Assert the HTTP status code of a Response.
+     *
+     * If the response is not as expected we set the assertion message to the
+     * body of the response - if it is json-decodable then we pretty print
+     * JSON.
+     *
+     * The $debugLength argument limits the number of lines included from the
+     * response body in case of failure.
+     */
+    protected static function assertHttpStatusCode(int $code, Response $response, int $debugLength = 50): void
+    {
+        $httpCode = $response->getStatusCode();
+
+        $message = '';
+        if ($code !== $httpCode) {
+            $message = $response->getContent();
+
+            if ('null' !== ($json = json_encode(json_decode($message, true), JSON_PRETTY_PRINT))
+                && $json) {
+                $message = explode(PHP_EOL, $json);
+            } else {
+                $message = explode(PHP_EOL, $message);
+            }
+
+            $message = implode(PHP_EOL, \array_slice($message, 0, $debugLength));
```

Comment:
If I personally would change anything in this function, it would be to remove this `debugLength`... If something is going wrong on CI I am always forced to do a rerun, because I might not have enough information.

Reply:
Here I'm not with you. I had the problem the other way around I was always struggleing when the debugLength was not keep in mention for HTML responses and did spam the whole CI/Terminal with unnecessary outputs.